flickr.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from logging import logging
+import os
+
+key = os.environ.get('flickr_key')
+
+def getImage(yelp_dish):
```

Comment:
may want variables in snake_case

Reply:
@NateBigStone  what is the snake_case?

Reply:
@xm6264jz [python_uses_snake_case](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) 🐍

Reply:
resolving convo